iothub/service/src/ImportDevicesRequest.cs

```diff
@@ -1,15 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace Microsoft.Azure.Devices
-{
-    /// <summary>
-    /// Used to provide parameters when requesting an import through the Resource Provider.
-    /// </summary>
-    public sealed class ImportDevicesRequest
```

Comment:
Unreferenced class that looks to be related to the import devices job.

Yes, it is a public class, so technically part of the published API. Should I just tag it as obsolete, instead?

As it is not referenced anywhere, likely, no one was using it.